src/main/java/seedu/address/logic/commands/DeleteTeamCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.TeamCommandsParser;
+import seedu.address.model.Model;
+import seedu.address.model.team.Team;
+
+public class DeleteTeamCommand extends Command {
+    public static final String COMMAND_WORD = "delete";
```

Comment:
There should be no collision since the command word is parsed as a subcommand, as in `team 1 delete`. However I agree that it might be confusing to have two different commands sharing the same word, and I'll probably move all the team subcommands to their own separate package to fix this.